instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/builder/internal/DefaultHttpClientInstrumenterBuilder.java

```diff
@@ -39,11 +41,18 @@
  */
 public final class DefaultHttpClientInstrumenterBuilder<REQUEST, RESPONSE> {
 
+  // copied from PeerIncubatingAttributes
+  private static final AttributeKey<String> PEER_SERVICE = AttributeKey.stringKey("peer.service");
+
   private final String instrumentationName;
   private final OpenTelemetry openTelemetry;
 
   private final List<AttributesExtractor<? super REQUEST, ? super RESPONSE>> additionalExtractors =
       new ArrayList<>();
+  private Function<
+          SpanStatusExtractor<? super REQUEST, ? super RESPONSE>,
+          ? extends SpanStatusExtractor<? super REQUEST, ? super RESPONSE>>
```

Comment:
```suggestion
  private Function<
          SpanStatusExtractor<REQUEST, RESPONSE>,
          ? extends SpanStatusExtractor<? super REQUEST, ? super RESPONSE>>
```

Reply:
no, 

`Function<? super SpanStatusExtractor<? super REQUEST, ? super RESPONSE>, ? extends SpanStatusExtractor<? super REQUEST, ? super RESPONSE>>`

or

`Function<? extends SpanStatusExtractor<? super REQUEST, ? super RESPONSE>, ? extends SpanStatusExtractor<? super REQUEST, ? super RESPONSE>>`

are closest, but neither work

this is the key

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/5edb8a1f10381a2867817728fc652e1dd9eb9c19/instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/builder/internal/DefaultHttpClientInstrumenterBuilder.java#L204